src/servers/trigger.ts

```diff
@@ -137,6 +142,39 @@ export async function onchainTransactionEventHandler(tx) {
   }
 }
 
+export async function onchainBlockEventhandler({ lnd, height }) {
+  // get incoming txs from the last 36 blocks ~6 hours
+  // we don't use default look back because daily cron job will take care of delayed transactions
```

Comment:
not sure to understand the comment about delayed transactions?

the cronjob is here in case the trigger pod is not running while a block get mined.

Reply:
removed